tests/testthat/test-uploadPackage.R

```diff
@@ -0,0 +1,84 @@
+
+test_that("Uploaded packages show up in summary table", {
+  # delete app DB if exists to ensure clean test
+  if (file.exists(test_path("test-apps", "database.sqlite"))) {
```

Comment:
We have a `skeleton.sqlite` file in `inst/testdata`. It might be faster to initialize `database.sqlite` from that file. Robert has done that in the unit tests he added. This can be done in a different PR though.